tapir/core/tests/test_email_preferences.py

```diff
@@ -17,36 +18,51 @@ def get_name(cls) -> str:
         return "VoluntaryTestMail"
 
 
-class TestVoluntaryMails(TapirFactoryTestBase):
+class TestOptionalNotDefaultMail(TapirFactoryTestBase):
     def setUp(self):
-        TapirEmailBase.register_email(VoluntaryMail)
+        TapirEmailBase.register_email(OptionalNonDefaultMail)
 
     def test_userWantsToOrHasToReceiveMail_mailNotSubscribedTo_shouldNotReceiveMail(
         self,
     ):
         tapir_user: TapirUser = TapirUserFactory.create()
 
         self.assertTrue(
-            VoluntaryMail().user_wants_to_or_has_to_receive_mail(user=tapir_user)
+            OptionalNonDefaultMail().user_wants_to_or_has_to_receive_mail(
+                user=tapir_user
+            )
             is False
         )
 
     def test_userWantsToOrHasToReceiveMail_mailSubscribedTo_shouldReceiveMail(self):
         tapir_user: TapirUser = TapirUserFactory.create()
-        tapir_user.additional_mails = ["tapir.coop.voluntaryTestMail"]
-
+        m = MailChoice(name=OptionalNonDefaultMail.get_unique_id(), choice=True)
+        m.save()
+        OptionalMails.objects.create(user=tapir_user, mail=m)
+        print(f"TESSSSSST: {tapir_user.optional_mails_by_user()}")
         self.assertTrue(
-            VoluntaryMail().user_wants_to_or_has_to_receive_mail(user=tapir_user)
+            OptionalNonDefaultMail().user_wants_to_or_has_to_receive_mail(
+                user=tapir_user
+            )
         )
 
-    def test_userWantsToOrHasToReceiveMail_mailIsMandatory_shouldReceiveMail(self):
+    # def test_userWantsToOrHasToReceiveMail_mailIsMandatory_shouldReceiveMail(self):
+    #     tapir_user: TapirUser = TapirUserFactory.create()
+    #     self.assertTrue(TapirAccountCreatedEmail.optional)
+    #
+    #     self.assertTrue(
+    #         TapirAccountCreatedEmail(
+    #             tapir_user=tapir_user
+    #         ).user_wants_to_or_has_to_receive_mail(user=tapir_user)
+    #     )
+
+    def test_mailIsChangedToNotEnabledByDefault(self):
```

Comment:
Let's keep the usual test naming convention of `test_functionBeingTested_case_expectedResult`